### ISSUE TITLE
OLW on a high density monitor shows very small text

### ISSUE THREAD

**Issue by smithkl42** [OpenLiveWriter/OpenLiveWriter#202]
See the attached screenshot. I have a Macbook Pro with a Retina screen, running Bootcamp. The text as it shows up in Open Live Writer is unusably small. My assumption is that the problem is that OLW is getting confused by the high-density display.

![image](https://cloud.githubusercontent.com/assets/2381026/11923757/eb5d4504-a75c-11e5-910f-dafd1e06e5db.png)

I checked, and didn't see any other issues open for this problem, but I could certainly have missed them.

**Comment by lextm** [comment]
Might be related to #44 .

**Comment by ScottIsAFool** [comment]
That and the font size in Live Writer is showing as 5.8, but in Word it's showing as 11... The font size will be based on what it has got from your blog's theme. If you set it to 11 in Live Writer, does it then look the same as Word?

**Comment by smithkl42** [comment]
@ScottIsAFool - Good point. On my desktop, which has just a regular monitor, the default font size shows up as 11.5. But when I publish from the machine with "5.8", it changes the 5.8 to the default font size, as you can see.

This:
![image](https://cloud.githubusercontent.com/assets/2381026/11931726/b879c80e-a7a4-11e5-965c-a7b8c7a322e1.png)

Becomes this:
![image](https://cloud.githubusercontent.com/assets/2381026/11931718/9718e1fe-a7a4-11e5-8ac1-245800ea0c1e.png)

**Comment by ScottIsAFool** [comment]
Ok, so on a high DPI monitor, it's setting the default font size in the app incorrectly?

**Comment by smithkl42** [comment]
Correct. It also appears to be setting most of the font sizes incorrectly, e.g., Heading2 is 4.5 and should be 9, Heading3 is 9 and should be 18. Oddly enough, it gets Heading1 correct at 24.

**Comment by ScottIsAFool** [comment]
Ok, thanks, that helps clarify what the bug is :)

**Comment by DawnL43072** [comment]
when writing my post in OLW I have my font size set at 12 (Georgia), but when it goes live it shrinks down to 9 or so! 
also picture orientation shifts about, not as in the original.

**Comment by gardnerjr** [comment]
as a workaround, i use ctrl+mousewheel (or pinch) to zoom in so that the tiny fonts are a normal size.  not sure if everyone knows about this since there's no view menus or zoom stuff in any toolbars or anywhere?

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
